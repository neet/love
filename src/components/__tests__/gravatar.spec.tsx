import * as React from 'react';
import * as ReactTestRenderer from 'react-test-renderer';
import Gravatar from '../gravatar';

describe('Gravatar', () => {
  it('renders a avatar component with given props', () => {
    const tree = ReactTestRenderer.create(
      <Gravatar
        className='class'
        email='n33t5hin@gmail.com'
        title='Neetshin'
        size={20}
        defaultImage='https://link-to-default-image.com'
        forceDefault={false}
        rating='g'
      />,
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
