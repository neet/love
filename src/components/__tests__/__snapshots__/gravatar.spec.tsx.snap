// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Gravatar renders a avatar component with given props 1`] = `
<img
  alt="Neetshin"
  className="class"
  src="https://secure.gravatar.com/avatar/edb1b2a09bf17af58dbabbabc1e0dddd?s=20px,d=https://link-to-default-image.com,f=false,r=g"
/>
`;
